src/components/Incorporation/ShareStructure.vue

```diff
@@ -124,27 +137,53 @@
                       />
                     </v-col>
                   </v-row>
-                  <v-label id="lbl-no-par" v-else>No par value</v-label>
+                  <v-label
+                    v-else
+                    id="lbl-no-par"
+                  >
+                    No par value
+                  </v-label>
                 </div>
 
                 <v-divider class="mt-2 mb-4" />
 
                 <div class="form__row">
                   <v-checkbox
                     id="special-rights-check-box"
-                    :label="'This share ' + shareStructure.type.toLowerCase() + ' has special rights or restrictions'"
                     v-model="shareStructure.hasRightsOrRestrictions"
+                    :label="'This share ' + shareStructure.type.toLowerCase() + ' has special rights or restrictions'"
```

Comment:
In case we develop anything like this in the future, there's no point in mixing strings and a function here -- it would be cleaner with just a function to return the whole label.